root/static/partials/sample/search/full-page.html

```diff
@@ -86,7 +86,8 @@
         </td>
         <td><a vv-href="/sample/{{ sample.sample_id }}">{{ sample.patient }}</a></td>
         <td>{{ sample.tissue_type }}</td>
-        <td>{{ sample.sample_date }}</td>
+        <td ng-if="!search.scientist.censor_dates || sample.derivation_protocol_id">{{ sample.sample_date }}</td>
+        <td ng-if="search.scientist.censor_dates && !sample.derivation_protocol_id">{{ sample.relative_date }}</td>
```

Comment:
Is the data still being sent to the client and only being filtered out by Angular?

Reply:
Yes.